lib/reports/notifications_from_notify.rb

```diff
@@ -0,0 +1,50 @@
+module Reports
+  class NotificationsFromNotify
+    def initialize(config: EmailAlertAPI.config.notify)
+      api_key = config.fetch(:api_key)
+      base_url = config.fetch(:base_url)
+
+      @client = Notifications::Client.new(api_key, base_url)
```

Comment:
I wonder if we can move this into EmailAlertApi.config so we can reuse the same initialisation in the NotifyProvider.